app/views/create/fromimage.html

```diff
@@ -298,12 +298,12 @@
                               >
                               <div class="form-group">
                                 <label for="scale-type">Strategy</label>
-                                <select ng-model="scaling.autoscale"
-                                        ng-options="option.value as option.label for option in scaling.autoscaleOptions"
-                                        id="scale-type"
-                                        class="form-control"
-                                        aria-describedby="scale-type-help">
-                                </select>
+                                <ui-select ng-model="scaling.autoscale" input-id="scale-type" aria-describedby="scale-type-help">
```

Comment:
I think the `aria-describedby` attributes need to go on the `form-group` element. I don't see them in the markup after the control is rendered.